python/cuml/fil/fil.pyx

```diff
@@ -573,6 +563,10 @@ class ForestInference(Base,
               maximum supported number of threads per GPU. Even if successful,
               it is not guaranteed that blocks_per_sm blocks will run on an SM
               concurrently.
+        forest_shape_file : file, io.StringIO or alike (default=None)
```

Comment:
Optional: if you use `**kwargs` for these functions and avoid duplicating parameters and documentation, you'll have to add `forest_shape_file` only once.

Reply:
let me fix the documentation